articles/app-service/configure-authentication-provider-openid-connect.md

```diff
@@ -48,6 +48,13 @@ If you are unable to use a configuration metadata document, you will need to gat
 1. Specify an application setting name for your client secret. Your client secret will be stored as an app setting to ensure secrets are stored in a secure fashion. You can update that setting later to use [Key Vault references](./app-service-key-vault-references.md) if you wish to manage the secret in Azure Key Vault.
 1. Press the **Add** button to finish setting up the identity provider. 
 
+> [!NOTE]
+> The OpenID provider name cannot contain symbols like "-" because an appsetting will be created based this and it does not support it. Use "_" instead.  
+
+> [!NOTE]
+> Azure requires “openid”, “profile” and “email” scopes, make sure you have configured your App Registration in your ID Provider with at least these scopes.  
+>
```

Comment:
```suggestion
> [!NOTE]
> The OpenID provider name can't contain symbols like "-" because an appsetting will be created based on this and it doesn't support it. Use "_" instead.  

> [!NOTE]
> Azure requires "openid," "profile," and "email" scopes. Make sure you've configured your App Registration in your ID Provider with at least these scopes.  
```